openedx/features/idea/forms.py

```diff
@@ -51,7 +53,7 @@ def __init__(self, *args, **kwargs):
         initial_arguments = kwargs.get('initial', {})
         self.user = initial_arguments.get('user', None)
         self.disable_field('organization_name', initial_arguments)
-        self.disable_field('country_name', initial_arguments)
+        self.disable_field('country', initial_arguments)
```

Comment:
This is not the name of country field in the model but key in initial_arguments dict.
Either we can revert it back to `country_name` or Update it at `edx-platform/openedx/features/idea/views.py:51` as well

We need to update it in test as well at `edx-platform/openedx/features/idea/tests/test_idea_views.py`

Reply:
Good catch! Done